packages/types/src/types.ts

```diff
@@ -114,7 +114,7 @@ export interface SlidevData {
 }
 
 export interface SlidevPreparserExtension {
-  name: string
+  name?: string
```

Comment:
I wasn't sure about this one, but I think `name` is optional. At least there's no type error and the example doesn't use a `name` either.